msi/release.sh

```diff
@@ -87,6 +90,6 @@ wix/light.exe \
 	obj/GitProduct.wixobj \
 	obj/GitComponents.wixobj \
 	-o $TARGET -ext WixUtilExtension \
-	-b / -b ../installer -sval &&
+	-b / -b ../installer &&
```

Comment:
Sorry, it not my goal to make you feel stupid but it also isn't my goal to create a primer how to use the WiX Toolset via commit messages.

The [linker documentation][doc] shows:

> -sval  Suppress MSI/MSM validation.

Validation is Windows Installer's term for "static analysis" (I tend to use the term static analysis now since more people know what that means). Static analysis is great at catching errors before the code is shipped.

It was disabled.

I enabled it because per-user folders are a great place to introduce mistakes that static validation will catch.

So that's what I was trying to say with:

> Also enables static analysis to catch errors handling per-user folders.

[doc]: http://wixtoolset.org/documentation/manual/v3/overview/light.html